py-utils/src/utils/msg_bus/message_bus/adaptee/kafka_python_adaptor.py

```diff
@@ -0,0 +1,69 @@
+from kafka import KafkaAdminClient, KafkaProducer, KafkaConsumer
```

Comment:
Lets use either confluent or kafka python. We have reviewed the comparison and we will only use one of them. WE need to limit about adding 3rd party dependencies in CORTX code.